app/request_parsers/mouse_event.py

```diff
@@ -33,13 +33,16 @@ class MouseEvent:
     relative_x: int
     relative_y: int
 
+    vwheel: int
```

Comment:
Can this be `vertical_wheel_delta` and `horizontal_wheel_delta`?

(Ditto throughout)